web-app/views/game.hbs

```diff
@@ -6,31 +6,25 @@
     <title>Hex</title>
     <link rel="stylesheet" type="text/css" href="/css/style.css" />
     <link rel="stylesheet" type="text/css" href="/css/game.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
   </head>
   <body>
     <div class="gameboard">
       <div class="box-header">
         {{#areEqual game.state.winner null}}
           {{#areEqual readyToPlay true}}
-            <p>
-              Current player:
-              <span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span>
-            </p>
+            {{#areEqual currentPlayerTurnToPlay true}}
+            <p><span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span>, your move!</p>
+            {{/areEqual}}
             {{#areEqual currentPlayerTurnToPlay false}}
-              <p>Waiting for other player to make their move...</p>
+              <p>Waiting <span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span> to make their move...</p>
             {{/areEqual}}
           {{else}}
             <p>Awaiting player 2...</p>
             <p>
               Send <a href="/games/{{game.id}}/join">this link</a> for second player to join
             </p>
```

Comment:
Because of these lines, we still have the flickering problem at the beginning of the game. I'll look into that as soon as I can 🙂

Reply:
Since it's only at the beginning of the game, it's not that big of a deal. I believed we can postpone this...